src/client/components/LeaderboardList/TeamAvatar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function TeamAvatar({ avatar }) {
+  if (!avatar) {
+    return (
+      <div className="avatar-div">
+        <div>avatar URL requierd</div>
```

Comment:
*required* 
Is the user able to enter the URL for the avatar? Or if there's no avatar the user should be redirected to the Avatar's menu?

Reply:
user should not see this div, i just put it for us in case of an empty string

Reply:
here I think its better if you write something as : ( Something wrong when fetching Avatar)

Reply:
but it's not allowed to user choose avatar out of what we have in the App.